### PR TITLE
Read me update

### PR DESCRIPTION
**Pull request by amyeallen1**
This PR incorporates a minor suggested correction to the instructions in the ReadMe on running simulations with JModelica, regarding where the jmodelica.py file is copied. Thanks, Nick! @nllong

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -94,9 +94,9 @@ desired, a user can run the simulations manually using JModelica (via Docker). F
 the runner to work locally.
 
 * Make sure jm_ipython.sh is in your local path.
-* After running the :code:`py.test` go into the :code:`tests/model_connectors/output` directory
-* Copy jmodelica.py to path :code:`cp ../../../geojson_modelica_translator/modelica/lib/runner/jmodelica.py .`
-* Run examples using either of the the following:
+* After running the :code:`py.test`, go into the :code:`../../../geojson_modelica_translator/modelica/lib/runner/` directory.
```
Comment (nllong) [inline README.rst:97]:
```suggestion
* After running the :code:`py.test`, go into the :code:`geojson_modelica_translator/modelica/lib/runner/` directory.
```

**Comment by nllong** [comment]
Hey @amyeallen1, it looks like the precommit failed on this last commit. Can you make sure to run the "pre-commit" command to remove the trialing whitespace:

https://github.com/urbanopt/geojson-modelica-translator#developers

Thanks!

**Comment by amyeallen1** [comment]
@nllong Thanks for the reminder! I ran pre-commit and pushed a commit with the changes. I think the failing tests now are due to unit tests--is that expected?

**Comment by nllong** [comment]
> @nllong Thanks for the reminder! I ran pre-commit and pushed a commit with the changes. I think the failing tests now are due to unit tests--is that expected?

Yes, these failed unit tests are part of the technical debt that we are incurring.

**Comment by amyeallen1** [comment]
Roger that. Thanks, Nick! @nllong

**Pull request merged**